packages/playground/src/components/deposit_dialog.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-container>
-    <v-dialog transition="dialog-bottom-transition" max-width="1000" v-model="depositDialog" persistent>
+    <v-dialog transition="dialog-bottom-transition" max-width="1000" v-model="depositDialog">
```

Comment:
when closing the dialog without pressing the close button, it doesn't open again

https://github.com/threefoldtech/tfgrid-sdk-ts/assets/41957921/4ff4af3a-fae1-4ec9-bfba-c1918d6a57ec

Reply:
Fixed.

Reply:
now it closes if we press `Esc`, click outside, or click the close button, but the event listener is still working in the background.

- try to open and close the deposit dialog 10 times
- wait for 2 minutes
- you will find 10 toasts as each dialog starts a listener and they are not destroyed.

Reply:
I reverted destroy changes and it should be working fine now.